Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,11 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  default_formula "libfuse"
```

Comment:
This is a Linuxbrew formula.

Since, unlike with the Java requirement, the relevant formula only supports Linux, this is placed here rather than in the OS-agnostic file.

I've made the assumption that if you're using brew and it's NOT on a Mac, you're using Linuxbrew's homebrew-core. Is that a reasonable assumption to make?